impeller/renderer/backend/vulkan/driver_info_vk.cc

```diff
@@ -317,6 +317,12 @@ void DriverInfoVK::DumpToLog() const {
   FML_LOG(IMPORTANT) << stream.str();
 }
 
+bool DriverInfoVK::CanBatchSubmitCommandBuffers() const {
+  return vendor_ == VendorVK::kARM ||
+         (adreno_gpu_.has_value() &&
+          adreno_gpu_.value() <= AdrenoGPU::kAdreno702);
```

Comment:
This is **not** how I would have groked this give the comment "Early 600 series Adreno drivers would deadlock".  This reads "adreno_gpu < 702" which sounds like 600 would pass.  AdrenoGPU enum should have probably been reverse sorted so the older gpu's are lower in value.

Specifically: can we make this check read better without having to know the order of AdrenoGPU?

Reply:
So for whatever reason, going in order from newest to oldest made sense to me at the time (two weeks ago) but on reflection, yeah writing it out is super confusing.

I will reverse the order of the enums so that later GPU models come ... later

Reply:
Reversed, thanks gemini